src/types.ts

```diff
@@ -17,12 +17,14 @@ type CommonOAuthAppStrategyOptions = {
   clientId: string;
   clientSecret: string;
   request?: OctokitTypes.RequestInterface;
+  onTokenCreated?: Function;
```

Comment:
ℹ️ Temporary types for draft PR

Reply:
If somebody sees an opportunity to have better inferred types, I'm open to suggestions